tutorials/vue/router-push.md

```diff
@@ -0,0 +1,93 @@
+[Vue Router](https://router.vuejs.org/installation.html) has two approaches to navigating a site,
+[declarative navigation](/tutorials/vue/router) and programmatic navigation which is shown below:
+
+```javascript
+const router = new VueRouter({
+    routes: [
+      {
+        path: '/home',
+        component: { template: '<h1>Home</h1>' }
+      },
+      {
+        path: '/about',
+        component: { template: '<h1>About Us</h1>' }
+      }
+    ]
+  });
+  
+  const app = new Vue({
+    router,
+    methods: {
+        changeRoute(route) {
+            router.push(route);
+        }
+    },
+    template: `
+      <div id="rendered-content">
+        <div>
+          <button @click="changeRoute('home')"> Home </button>
+          <button @click="changeRoute('about')"> About us </button>
```

Comment:
Why the space after `>` and before `About`, and then after `us` and before `</`?